index.html

```diff
@@ -11,14 +11,14 @@
 
         <footer>
             <div class="footer-nav">
-                <a href="#">
-                    <botton>🏂</botton>
+                <a href="index.html">
+                    <botton class="btn btnactive">🏂</botton>
```

Comment:
Typo:
It should be `button` instead of `botton`.
This happens multiple times

Reply:
Don't drink and drive.. seems to be true 🍺 thx :)